src/GeneralStateTestsFiller/stCreate2/CREATE2_FirstByte_loopFiller.yml

```diff
@@ -0,0 +1,59 @@
+# The test calls CREATE2 in a loop deploying 1-byte contracts with all possible byte values, records in storage the values that failed to deploy.
+CREATE2_FirstByte_loop:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 100000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul
+        {
+          // { mstore8(0, 0x00) return(0, 1) }
```

Comment:
this is the contract constructor code.

Reply:
Yes, intention is to create contracts with conde that consists of one byre

Reply:
Added some comments in yul code.